server/db/User.js

```diff
@@ -1,4 +1,5 @@
 const { Model } = require('sequelize');
+const office = '[a-z][0-9]{1,2}$';
```

Comment:
This is a string, not a regex. It should be regex.
Also, I like the convention to prefix regexes with re so rename it to `reOffice`.